lib/eventsource.js

```diff
@@ -471,3 +470,16 @@ function removeUnsafeHeaders (headers) {
 
   return safe
 }
+
+/**
+ * Transform an URL to a valid origin value.
+ *
+ * @param {String|Object} url URL to transform to it's origin.
+ * @returns {String} The origin.
+ * @api private
```

Comment:
should leave `@link https://github.com/unshiftio/original/blob/507b362269c0ad4405d095aedc227c40aecaf68a/index.js` as credit to orignal code.